questions/shop.md

```diff
@@ -0,0 +1,104 @@
+# How to make a shop
+This is a quick guide of how to make a shop where you can buy items with other items or costing a scoreboard value (money or coins).
+
+## Item shop
+This method consists of buying items with other items, in this example, you will buy 1 netherite ingot with 5 diamonds.
+
+> [!NOTE]
+> You can add any command you wan't to run (for example a playsound) before the last command, but the selector must be the same as the command before, (`@a[tag=buy_netherite]` in this case)
+
+### Java
+There are 2 methods, one for pre-1.20.5 and the other for 1.20.5 and above, this is due to the new `execute if items` subcommand.
+> [!NOTE]
+> The method for pre-1.20.5 will still work in 1.20.5 and above.
+
+Both methods will require this setup:
+
+    # In chat / Load function
+    scoreboard objectives add diamonds dummy
+
+Both methods works the same:
+First we are going to know how many diamonds the player has storing the value in a scoreboard and if the player has that that number of diamonds we will add a tag (first and second command) or we will run a function instead if using a datapack.
+After that we are going to give the netherite ingot to the player with the tag and we will clear the diamonds from it.
+To avoid the player from a loop we will remove the tag as we dont need it more.
+
+#### Pre-1.20.5
+
+    # Command blocks
+    execute as @p store result score @s diamonds run clear @s diamond
+    execute as @p at @s[scores={diamonds=5..}] run tag @s add @s buy_netherite
+    give @a[tag=buy_netherite]  netherite_ingot 1
+    clear @a[tag=buy_netherite] diamond 5
+    tag @a remove buy_netherite
+
+Or if you prefer a function:
+
+    # function example:load
+    scoreboard objectives add diamonds dummy
+    
+    # function example:buy/netherite
+    execute store result score @s diamonds run clear @s diamond 0
+    execute if entity @s[scores={diamonds=5..}] run function example:buy/netherite/success
+
+    # function example:buy/netheirte/success
+    give @s netherite_ingot 1
+    clear @s diamond 5
+    ## you can add any command you want here
```

Comment:
It's worth adding a mention that you need to run function `example:buy/netherite` `as` the player who makes the purchase.